largest-contentful-paint/image-upscaling.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>Largest Contentful Paint: largest image is reported.</title>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/largest-contentful-paint-helpers.js"></script>
+<script src="/common/utils.js"></script>
+<script>
+    setup(() =>
+        assert_implements(window.LargestContentfulPaint, "LargestContentfulPaint is not implemented"));
+
+    const imageURL = `${window.location.origin}/images/blue.png`;
+    async function load_image_and_get_lcp_size(t, imageStyle = {}, containerStyle = {}) {
+        const popup = window.open('about:blank');
+        t.add_cleanup(() => popup.close());
+        const image = popup.document.createElement('img');
+        image.src = imageURL;
+        await image.decode();
```

Comment:
I feel this needs some comments to explain why we are creating so many different things

Reply:
Added a couple of comments